src/java.base/share/classes/jdk/internal/io/JdkConsoleProvider.java

```diff
@@ -31,15 +31,15 @@
  * The provider used for instantiating JdkConsole instance can be
  * specified with the system property "jdk.console", whose value
  * designates the module name of the implementation, and which defaults
- * to "jdk.internal.le" (jline). If no providers is available,
+ * to "java.base". If no providers is available,
  * or instantiation failed, java.base built-in Console implementation
```

Comment:
The overall change looks fine but I think for the next edit that we should move most of this comment to Console as it's Console that selects the behavior and that skips errors. Also once the SM execution mode goes away then we can re-visit that behavior.

Reply:
Makes sense. The comment has been moved to `Console`.